webcheck.go

```diff
@@ -188,23 +188,49 @@ func (fm *Frontman) runWebCheck(data WebCheckData) (m map[string]interface{}, er
 		}
 
 	} else {
-		b := make([]byte, 512)
-		for true {
-			n, err := resp.Body.Read(b)
-			totalBytes += n
-			if err != nil {
-				break
-			}
+		_, err = ioutil.ReadAll(resp.Body)
```

Comment:
Why do we read the body if we don't use it?

Reply:
To count the size.
It is done in the underlying ReadCloser (ReadCloserCounter)